bower.json

```diff
@@ -21,7 +21,8 @@
     "tests"
   ],
   "dependencies": {
-    "polymer": "^1.8.0"
+    "polymer": "^1.8.0",
+    "paper-input": "PolymerElements/paper-input#^1.1.23"
```

Comment:
This is the dependency for `paper-textarea`:` paper-input/paper-textarea.html`
Will add paper-button and iron-icon too.